tests/nlu/featurizers/test_lm_featurizer.py

```diff
@@ -1,21 +1,49 @@
 import os
-from typing import Text, List, Dict, Tuple
+from typing import Text, List, Dict, Tuple, Any, Callable
 
 import numpy as np
 import pytest
 import logging
 
 from _pytest.logging import LogCaptureFixture
 
+from rasa.engine.graph import ExecutionContext
+from rasa.engine.storage.storage import ModelStorage
+from rasa.engine.storage.resource import Resource
 from rasa.nlu.constants import (
     TOKENS_NAMES,
     NUMBER_OF_SUB_TOKENS,
 )
 from rasa.nlu.tokenizers.whitespace_tokenizer import WhitespaceTokenizer
 from rasa.shared.nlu.training_data.training_data import TrainingData
 from rasa.shared.nlu.training_data.message import Message
-from rasa.nlu.featurizers.dense_featurizer.lm_featurizer import LanguageModelFeaturizer
+from rasa.nlu.featurizers.dense_featurizer.lm_featurizer import (
+    LanguageModelFeaturizerGraphComponent,
+)
 from rasa.shared.nlu.constants import TEXT, INTENT
+from rasa.nlu.tokenizers.tokenizer import Token
+
+
+@pytest.fixture
+def resource_language_model_featurizer() -> Resource:
+    return Resource("LanguageModelFeaturizerGraphComponent")
+
+
+@pytest.fixture
+def create_language_model_featurizer(
+    default_model_storage: ModelStorage,
+    resource_language_model_featurizer,
+    default_execution_context: ExecutionContext,
+) -> Callable[[Dict[Text, Any]], LanguageModelFeaturizerGraphComponent]:
+    def inner(config: Dict[Text, Any]):
+        return LanguageModelFeaturizerGraphComponent.create(
+            config=config,
```

Comment:
```suggestion
            config={**LanguageModelFeaturizerGraphComponent.get_default_config(), **config},
```

Reply:
I do this in the [constructor](https://github.com/RasaHQ/rasa/blob/6e12ff8a6fdaff3ac1c0de86a55a8b69d5cedb38/rasa/nlu/featurizers/dense_featurizer/lm_featurizer.py#L70). Do I need it here too?

Reply:
oooh. Doesn't need to be done in the constructor as the graph will do this on the outside (see [here](https://github.com/RasaHQ/rasa/blob/5bb331ff7ba75ad564478cbc71fd1a46bce5ee02/rasa/engine/graph.py#L316))